wca-regulations.md

```diff
@@ -507,6 +507,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
     - H1a) Before an attempt, the competitor must submit to the organization team the number of puzzles (at least 2) they wish to attempt.
         - H1a1) A competitor is not permitted to change the number of puzzles after submitting their number to the organization team.
         - H1a2) A competitor may request to keep their submission private until all competitors have submitted their numbers. Once all competitors have reported their numbers, the information becomes public (e.g. a competitor may ask for the numbers of any other competitors before starting their attempt).
+        - H1a3) Before the attempt, the scrambled puzzles must be placed on the table in an arbitrary orientation while ensuring they remain completely covered. The puzzles should be arranged in a preferably square shape (e.g. 9 puzzles should be arranged in 3 rows and 3 columns).
```

Comment:
The English here feels a little clunky to me, but I don't know exactly how to fix it. I think it would help if the word "preferably" were moved earlier in the sentence (maybe before "should"). Another phrasing that feels less binary to me: "should be arranged in shape as square as possible".

Reply:
Yes, that sounds nicer. Thanks for the suggestion!
I updated the PR.